w3af/plugins/crawl/soap.py

```diff
@@ -0,0 +1,168 @@
+import contextlib
+import sys
+from cStringIO import StringIO
+
+import zeep
```

Comment:
Did you add this to the requirements file(s)?

Reply:
fixed